data/scripts/actions/objects/cask_and_kegs.lua

```diff
@@ -1,60 +1,70 @@
-local targetIdList = {
-	[25879] = { itemId = 285, transform = 266 },
-	[25880] = { itemId = 283, transform = 236 },
-	[25881] = { itemId = 284, transform = 239 },
-	[25882] = { itemId = 284, transform = 7643 },
-	[25883] = { itemId = 284, transform = 23375 },
-	[25889] = { itemId = 285, transform = 268 },
-	[25890] = { itemId = 283, transform = 237 },
-	[25891] = { itemId = 284, transform = 238 },
-	[25892] = { itemId = 284, transform = 23373 },
-	[25899] = { itemId = 284, transform = 7642 },
-	[25900] = { itemId = 284, transform = 23374 },
-	[25903] = { itemId = 285, transform = 266 },
-	[25904] = { itemId = 283, transform = 236 },
-	[25905] = { itemId = 284, transform = 239 },
-	[25906] = { itemId = 284, transform = 7643 },
-	[25907] = { itemId = 284, transform = 23375 },
-	[25908] = { itemId = 285, transform = 268 },
-	[25909] = { itemId = 283, transform = 237 },
-	[25910] = { itemId = 284, transform = 238 },
-	[25911] = { itemId = 284, transform = 23373 },
-	[25913] = { itemId = 284, transform = 7642 },
-	[25914] = { itemId = 284, transform = 23374 },
+local targetTransformationList = {
+	[25879] = { flaskId = 285, transformId = 266 },
+	[25880] = { flaskId = 283, transformId = 236 },
+	[25881] = { flaskId = 284, transformId = 239 },
+	[25882] = { flaskId = 284, transformId = 7643 },
+	[25883] = { flaskId = 284, transformId = 23375 },
+	[25889] = { flaskId = 285, transformId = 268 },
+	[25890] = { flaskId = 283, transformId = 237 },
+	[25891] = { flaskId = 284, transformId = 238 },
+	[25892] = { flaskId = 284, transformId = 23373 },
+	[25899] = { flaskId = 284, transformId = 7642 },
+	[25900] = { flaskId = 284, transformId = 23374 },
+	[25903] = { flaskId = 285, transformId = 266 },
+	[25904] = { flaskId = 283, transformId = 236 },
+	[25905] = { flaskId = 284, transformId = 239 },
+	[25906] = { flaskId = 284, transformId = 7643 },
+	[25907] = { flaskId = 284, transformId = 23375 },
+	[25908] = { flaskId = 285, transformId = 268 },
+	[25909] = { flaskId = 283, transformId = 237 },
+	[25910] = { flaskId = 284, transformId = 238 },
+	[25911] = { flaskId = 284, transformId = 23373 },
+	[25913] = { flaskId = 284, transformId = 7642 },
+	[25914] = { flaskId = 284, transformId = 23374 },
```

Comment:
You could remove the flaskId and leave only the transformId, because in Tibia Global you don't have this flask distinction, You can use a great, strong or small flask in any of the cask, being able to replace the small to great or strong for example.

Reply:
I didn't do the tests, but I believe it complies with your recommendations.

Reply:
The first version from girl was already working perfectly; from there on, it's just formality and ego competition.